hipo_drf_exceptions/exceptions.py

```diff
@@ -1,7 +1,15 @@
+from django.utils.translation import ugettext_lazy as _
```

Comment:
`ugettext_lazy` is [deprecated](https://docs.djangoproject.com/en/3.0/releases/3.0/#id3) in Django 3.0 so it would be nice if we use `gettext_lazy` here for the sake of compatibility.

Reply:
Thank you, fixed.